src/components/HomeSectionClients/styled.ts

```diff
@@ -71,6 +85,19 @@ const Logo = styled.img`
   `};
 `;
 
+export const TagContainer = styled.div`
+  margin-top: 4px;
+
+  div {
```

Comment:
I’m doubtful about this approach to styling. `styled-components` are centered about the idea of making every tag a styled component. So this should probably be a separate `export const Tag` or something, not a `div`. What do you think?